### ISSUE TITLE
Error: Cannot find module 'webpack.next/lib/NormalModule' on v0.4.0

### ISSUE THREAD

**Issue by kirkcola** [pmmmwh/react-refresh-webpack-plugin#166]
Hi,

I was trying to integrate the latest version of this plugin (v0.4.0) with the lastest beta of webpack 5 (v5.0.0-beta.22) and got this error:

```
(node:9373) UnhandledPromiseRejectionWarning: Error: Cannot find module 'webpack.next/lib/NormalModule'
Require stack:
- /Users/kirk/test/node_modules/@pmmmwh/react-refresh-webpack-plugin/lib/index.js
- /Users/kirk/test/webpack/webpack.dev.js
- /Users/kirk/test/scripts/start.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:966:15)
    at Function.Module._load (internal/modules/cjs/loader.js:842:27)
    at Module.require (internal/modules/cjs/loader.js:1026:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at /Users/kirk/test/node_modules/@pmmmwh/react-refresh-webpack-plugin/lib/index.js:232:34
    at Hook.eval [as call] (eval at create (/Users/kirk/test/node_modules/webpack/node_modules/tapable/lib/HookCodeFactory.js:19:10), <anonymous>:200:1)
    at Hook.CALL_DELEGATE [as _call] (/Users/kirk/test/node_modules/webpack/node_modules/tapable/lib/Hook.js:14:14)
    at Compiler.newCompilation (/Users/kirk/test/node_modules/webpack/lib/Compiler.js:885:26)
    at /Users/kirk/test/node_modules/webpack/lib/Compiler.js:926:29
    at eval (eval at create (/Users/kirk/test/node_modules/webpack/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:16:1)
```

Sorry I am not familiar with this project, but i think maybe this is a typo? Because after I replaced the 'webpack.next' with 'webpack', everything worked just fine:

https://github.com/pmmmwh/react-refresh-webpack-plugin/blob/937ed77c6c8c70470a95bbed03bd280c650f0f34/lib/index.js#L232

Thanks.

**Comment by kirkcola** [comment]
I've create a PR to fix it #167

**Comment by pmmmwh** [comment]
Fixed by #167, released in 0.4.1.
